website/js/app.js

```diff
@@ -3,6 +3,13 @@ $(document).ready(function() {
     // init foundation
     $(document).foundation()
 
+    // If this flag is set, allow the user to kick off an Ansible job
+    window.rhtLabsInternal = (getQueryVariable('internal') === 'true');
+    if (window.rhtLabsInternal) {
+      // Persist internal flag when form is submitted.
+      $('#form').append('<input type="hidden" name="internal" value="true" />');
+    }
+
```

Comment:
There's also a check at the bottom of this file for `rhtLabsInternal` and sets the form action to `internal.html` - shouldn't that now be updated? I know `internal.html` redirects, but probably best to eliminate if possible?

Reply:
Thanks for catching that! Fixed.